test/ParserTest.php

```diff
@@ -41,6 +41,16 @@ public function parserProvider()
 
         $eqAST = new AST\EqOperation;
         $eqAST->setProperty('name');
+        $eqAST->setValue('"12"');
+        $tests['simple eq with numeric in double quotes'] = array('eq(name,"12")', $eqAST);
```

Comment:
I'm getting something wrong but am not what. Isn't this now ``name === '12'`` where name must be a string? Would ``eq(size,'12')`` result in ``size === '12'`` (ie. always false) when size is an int? Looking at other implementations it seems that rql doe implicit strings all the time.

What about ``eq(name,avec'apos)`` and ``eq(name,"Hans 'Housi' Wale-Sepp")``?